diga-epa-client/src/test/java/com/oviva/epa/client/KonnektorServiceAcceptanceTest.java

```diff
@@ -98,6 +98,35 @@ void writeDocument() {
     assertDoesNotThrow(() -> konnektorService.writeDocument(recordIdentifier, document));
   }
 
+  @Test
+  void replaceDocument() {
+
+    var documentId = UUID.randomUUID();
+
+    // when
+
+    // 1) get home community
+    var hcid = konnektorService.getHomeCommunityID(KVNR);
+    var recordIdentifier = new RecordIdentifier(KVNR, hcid);
+
+    // 2) read author/telematik ID from SMC-B
+    var authorInstitution =
+        konnektorService.getAuthorInstitutions().stream()
+            .findFirst()
+            .orElseThrow(() -> new IllegalStateException("no SMC-B found"));
+
+    var document = buildDocumentPayload(documentId, authorInstitution, EXPORT_XML.getBytes());
+
+    // 3) write the FHIR/MIO document
+    assertDoesNotThrow(() -> konnektorService.writeDocument(recordIdentifier, document));
+
+    // 4) replace the document
+    var newDocumentId = UUID.randomUUID();
+    var newDocument = buildDocumentPayload(newDocumentId, authorInstitution, EXPORT_XML.getBytes());
+    assertDoesNotThrow(
+        () -> konnektorService.replaceDocument(recordIdentifier, newDocument, documentId));
+  }
+
```

Comment:
did it actually work in the RU? the first time I tried I got an error, but that was weeks ago now

Reply:
Yes, that works. Repeatedly tried.